src/report/superset-oauth-login-modal.scss

```diff
@@ -0,0 +1,40 @@
+.superset-oauth-login-modal {
```

Comment:
That is right. It is based on auth-modal class. Reasons, why I have created the new class, are:
* auth-modal is placed in another repository, so it means that it could be difficult to find it in one IDE.
* some lines of auth-modal has been changed. 
I can leave it or try to override only changed styles. So, what should I do @mkwiatkowskisoldevelo?

Reply:
1. You can always put multiple repositories in one IDE, like Visual Code, it is not a good reason to duplicate code (even just styling)
2. Do those lines need to be changed? Do we really need this modal to look different than regular login modal? If so than just change those lines and use both CSS classes in this new modal.